features/04_aruba_api/core/expand_path.feature

```diff
@@ -69,7 +69,7 @@ Feature: Expand paths with aruba
     RSpec.describe 'Expand path', :type => :aruba do
       let(:path) { '~/path/to/dir' }
 
-      it { expect(expand_path(path)).to match %r</home/[^/]+/path/to/dir> }
+      it { expect(expand_path(path)).to match %r</home/([^/]+/)+path/to/dir> }
```

Comment:
No, I think, it was a "hack" I introduced. But you're right, the correct way was to remove that test. I missed the comment last night. Thanks.